paper-shadow.html

```diff
@@ -23,24 +23,80 @@
 -->
 
 <link href="../polymer/polymer.html" rel="import">
+<link href="../layout/layout.html" rel="import">
+<dom-module id="paper-shadow">
+  <style>
+    paper-shadow,
+    paper-animated-shadow {
+      display: block;
+      position: relative;
+    }
+
+    paper-shadow #shadow-bottom,
+    paper-shadow #shadow-top {
+      border-radius: inherit;
+      pointer-events: none;
+    }
+
+    paper-shadow #shadow-bottom[animated],
+    paper-shadow #shadow-top[animated] {
+      transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-1 {
+      box-shadow: none;
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-1 {
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-2 {
+      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2);
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-2 {
+      box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.3);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-3 {
+      box-shadow: 0 11px 7px 0 rgba(0, 0, 0, 0.19);
+    }
 
-<link href="paper-shadow.css" rel="stylesheet" shim-shadowdom>
+    paper-shadow #shadow-bottom.paper-shadow-z-3 {
+      box-shadow: 0 13px 25px 0 rgba(0, 0, 0, 0.3);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-4 {
+      box-shadow: 0 14px 12px 0 rgba(0, 0, 0, 0.17);
+    }
 
-<polymer-element name="paper-shadow">
+    paper-shadow #shadow-bottom.paper-shadow-z-4 {
+      box-shadow: 0 20px 40px 0 rgba(0, 0, 0, 0.3);
+    }
 
-<template>
+    paper-shadow #shadow-top.paper-shadow-z-5 {
+      box-shadow: 0 17px 17px 0 rgba(0, 0, 0, 0.15);
+    }
 
-  <div id="shadow-bottom" fit animated?="[[animated]]" class="paper-shadow-bottom-z-[[z]]"></div>
-  <div id="shadow-top" fit animated?="[[animated]]" class="paper-shadow-top-z-[[z]]"></div>
+    paper-shadow #shadow-bottom.paper-shadow-z-5 {
+      box-shadow: 0 27px 55px 0 rgba(0, 0, 0, 0.3);
+    }
+  </style>
+  <template>
 
-  <content></content>
+    <div id="shadow-bottom" animated$="[[animated]]" class$="[[paperShadowClassName]]"></div>
```

Comment:
Arguable but it seems like `animatable` changes the behavior of this element (and therefore uses an attribute), even though that behavior is implement with CSS. The reason this element exists is pretty much only to express a style, so whether it is animated or not seems part of its behavior.